resources/views/checkout/partials/sections/login.blade.php

```diff
@@ -1,34 +1,51 @@
-<login v-slot="{ email, password, go, loginInputChange, emailAvailable }">
+<login v-slot="{ email, password, go, loginInputChange, emailAvailable, logout }">
     <section>
-        <form id="login" class="grid gap-5 md:grid-cols-2" v-on:submit.prevent="go()">
+        <div class="grid gap-5 md:grid-cols-2">
+            <template v-if="!loggedIn">
+                <x-rapidez-ct::input
+                    label="E-mail address"
+                    name="email"
+                    type="email"
+                    v-bind:value="email"
+                    v-on:input="loginInputChange"
+                    v-on:blur="$root.guestEmail = email; if(!password) { go() }"
+                    required
+                    placeholder="Enter your e-mail address"
+                />
 
-            <x-rapidez-ct::input
-                label="E-mailaddress"
-                name="email"
-                type="email"
-                v-bind:value="email"
-                v-on:input="loginInputChange"
-                required
-                placeholder="Enter your e-mailaddres"
-            />
+                <x-rapidez-ct::input
+                    v-if="!emailAvailable"
+                    label="Password"
+                    name="password"
+                    type="password"
+                    ref="password"
+                    v-on:input="loginInputChange"
+                    required
+                />
 
-            <x-rapidez-ct::input
-                v-if="!emailAvailable"
-                label="Password"
-                name="password"
-                type="password"
-                ref="password"
-                v-on:input="loginInputChange"
-                required
-            />
+                <p v-if="!emailAvailable" class="self-end">Je hebt al een account met dit e-mailadres. Login om verder te gaan</p>
+                <p v-else class="self-end">Naar dit e-mailadres sturen wij uw bestelling bevestiging. Ook controleren we of u al een account hebt zodat u sneller kunt afrekenen.</p>
 
-            <p v-if="!emailAvailable" class="self-end">Je hebt al een account met dit e-mailadres. Login om verder te gaan</p>
-            <p else class="self-end">Naar dit e-mailadres sturen wij uw bestelling bevestiging. Ook controleren we of u al een account hebt zodat u sneller kunt afrekenen.</p>
-
-            <x-rapidez-ct::button.accent v-if="!emailAvailable" type="submit" dusk="continue">
-                @lang('Login')
-            </x-rapidez-ct::button.accent>
-        </form>
+                <x-rapidez-ct::button.accent v-if="!emailAvailable" dusk="continue" v-on:click.prevent="go">
+                    @lang('Login')
+                </x-rapidez-ct::button.accent>
+            </template>
+            <template v-else>
+                <div class="h-[52px] px-4 flex items-center rounded border border-border bg-inactive-200">
```

Comment:
can you prefix the border-color with ct- please

Reply:
Same with inactive-200

Reply:
border-border shouldn't be needed if the default border color is set within the checkout-theme config